ui/app/css/itcss/components/item.scss

```diff
@@ -0,0 +1,116 @@
+.Item__flex-grid {
+  width: 100%;
+  min-width: 374px;
+  min-height: 86px;
+  margin: 0 20px;
+
+  left: 0px;
+  top: 497px;
+  background: #FFFFFF;
+  padding: 24px 16px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  border-top: 1px solid #E5E5E5;
```

Comment:
In the interest of stepping towards a design system, can we add a variable for `#E5E5E5` to `variables.scss` and us the variable here instead? Our names for these variable has been arbitrary in the past (using http://chir.ag/projects/name-that-color/). We could now start using some sort of functional names.